worlds/trackmania/options.py

```diff
@@ -0,0 +1,319 @@
+from dataclasses import dataclass
+from datetime import datetime  # for custom_series: uploaded_before and uploaded_after
+from schema import Schema, And, Or, Optional  # for custom series validation
+from typing import List, Dict, Any
```

Comment:
```suggestion
from typing import Any
```
Both List and Dict can be replaced with their lowercase versions for typing (obviously would need to change all of the instances of them)